tests/integrate/tools/catch_properties.sh

```diff
@@ -400,3 +401,11 @@ fi
 ttot=`grep $word $running_path | awk '{print $3}'`
 echo "totaltimeref $ttot" >>$1
 
+if ! test -z "$symmetry" && [ $symmetry == 1 ]; then
```

Comment:
you should add these codes before `echo "totaltimeref $ttot" >>$1`

Reply:
The items after `totaltimeref` will not be checked

Reply:
OK, done.